### PR TITLE
Make endpoint group for observations API

### PR DESCRIPTION
**Pull request by shifucun**
There have been a lot of webdriver tests failure, which could be due to resource issue in autopush mixer. Create a new group for observations API to narrow down / fix the issue.

### CODE REVIEW

File: deploy/helm_charts/envs/mixer_autopush.yaml [hunk 1]
```diff
@@ -34,6 +34,17 @@ serviceGroups:
       resources:
         memoryRequest: "400M"
         memoryLimit: "400M"
+  - mixer-default-service:
```
Comment (Fructokinase) [inline deploy/helm_charts/envs/mixer_autopush.yaml:37]: mixer-observations-service?

File: deploy/helm_charts/mixer/values.yaml [hunk 2]
```diff
@@ -120,6 +120,17 @@ serviceGroups:
       resources:
         memoryRequest: "400M"
         memoryLimit: "400M"
+  - mixer-default-service:
```
Comment (Fructokinase) [inline deploy/helm_charts/mixer/values.yaml:123]: mixer-observations-service?

File: deploy/helm_charts/envs/mixer_prod.yaml [hunk 3]
```diff
@@ -11,16 +11,16 @@ ingress:
...
       urlPaths:
         - "/place/stat-var-group/*"
         - "/stat-var/*"
         - "/v1/info/variable/group/*"
         - "/v1/variable/*"
       replicas:
-        default: 30
-        min: 30
-        max: 50
+        default: 10
```
Comment (shifucun) [inline deploy/helm_charts/envs/mixer_prod.yaml:21]: These API is very low cost in memory key value read and should be serve high qps per pod, hence reducing the replicas.

File: deploy/helm_charts/envs/mixer_autopush.yaml (hunk 1 above)
Comment (shifucun) [reply deploy/helm_charts/envs/mixer_autopush.yaml]: updated all the name to use just the label.

**Review by Fructokinase — APPROVED** [review]

**Pull request merged**